websites/D/Di.FM/metadata.json

```
@@ -12,7 +12,7 @@
 	},
 	"url": "di.fm",
 	"regExp": "([a-z0-9-]+[.])*di[.]fm[/]",
-	"version": "1.4.13",
+	"version": "1.4.14",
```

This adds functionality so should be a minor bump
```suggestion
	"version": "1.5.0",
```